developer-docs/upgrade/build_n_deploy.md

```diff
@@ -7,11 +7,36 @@ allowSearch: true
 keywords: Upgrade, Sunbird 2.3.0, Core, elasticsearch
 ---
 
-## Overview
+**New load balancer setup for Swarm manager**
```

Comment:
@keshavprasadms Need an Overview for this file. The information structure of this file is not correct. We can discuss and close it.

Reply:
Added overview

Reply:
Ok